packages/lesmis/src/components/EntityVitalsItem.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ *
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import MuiContainer from '@material-ui/core/Container';
+import {
+  PinDrop,
+  People,
+  School,
+  LocalLibrary,
+  Assignment,
+  Map,
+  LocationOn,
+} from '@material-ui/icons';
+import Typography from '@material-ui/core/Typography';
+
+const Wrapper = styled.section`
+  padding-top: 0.5rem;
+  padding-left: 0;
+  padding-right: 0;
+  padding-bottom: 0;
+`;
+
+const Container = styled(MuiContainer)`
+  display: flex;
+  flex-direction: row;
+  width: 30%;
+  margin-left: 0;
+  margin-right: 0;
+`;
+
+const VitalName = styled(Typography)`
+  color: ${props => props.theme.palette.text.secondary};
+  font-size: 0.75rem;
+`;
+
+const VitalContent = styled(Typography)`
+  color: ${props => props.theme.palette.text.primary};
+  font-weight: bold;
+`;
+
+const StyledPinDrop = styled(PinDrop)`
+  padding-top: 0.5rem;
```

Comment:
To save repeating these styles on every icon, we can target the icon using its class from the parent component. See the suggestion above.